### PR TITLE
Chandler James homework submission

### PR DESCRIPTION
**Pull request by chandlerjames**
Thanks!

### CODE REVIEW

**Review by deblnia — COMMENTED** [review]
Evaluation HW-01
----------------------------------------------------
Chandler's biography is succinct and useful. He's also used some Markdown formatting (though he is one short of the required four). 

| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |              |         x   |
| **Coding strategy**         |           |   x           |            |
| **Presentation**            |           |         x     |            |
| **Achievement, creativity** |           |    x          |            |
| **Ease of access**          |           |     x         |            |

Remarks:

* I would've liked to see the link linked inline using Markdown syntax and more comments on the process of using Github (because that's what I'm trying to learn). 
* More content is always better! I would've liked to learn more about Chandler.

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |    x       |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |     x      |              |            |

Remarks:

* Clean, simple, uses different Markdown syntaxes
* No extraneous files in the repo
* Workflow properly followed

File: README.md [hunk 1]
```diff
@@ -1,2 +1,16 @@
...
+# Introduction
+
+Hello. My name is Chandler James. I'm a second year PhD student in the political science department. Before graduate school, I worked as a staffer for a US congressman. My research interests include:
+
+* American politics
+* institutions 
+* public opinion
+
+
+You can follow me on *twitter* at this link -> http://twitter.com/chnjms
```
Comment (elsygonzalez) [inline README.md:13]: This is great! If you're interested in taking linking to the next level, I got the comment in my work to wrap the text in `[]` and `()` to directly link the URL to a text. No necessary, but fun fact. 

For example:

`You can follow me on [*Twitter*](http://twitter.com/chnjms)`

You can follow me on [*Twitter*](http://twitter.com/chnjms)

**Comment by elsygonzalez** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     X      |              |            |
| **Coding strategy**         |      X     |              |            |
| **Presentation**            |      X     |              |            |
| **Achievement, creativity** |           |        X      |            |
| **Ease of access**          |     X      |              |            |

Remarks:

* Great use of markdown elements!
* A bit short, and could have gone on with more details, but you did fulfill the rubric of the assignment.
* If you're interested you could link the URL in rich text, but not necessary.
* Great job!

File: README.md (hunk 1 above)
Comment (elsygonzalez) [inline README.md:13]: Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      X     |              |            |
| **Coding strategy**         |      X     |              |            |
| **Presentation**            |     X      |              |            |
| **Achievement, creativity** |           |       X       |            |
| **Ease of access**          |     X      |              |            |

Remarks:
*Great use of markdown elements!
*A bit short, could have gone on with more details to fully showcase the elements, but you did fulfill the rubric of the assignment.
*If you're interested you could link the URL in rich text, but not necessary.
*Great job!

**Pull request closed without merging**